drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -708,6 +690,7 @@ function DragAndDropBlock(runtime, element, configuration) {
                         releaseItem($selectedItem);
                     } else if (isActionKey(evt)) {
                         evt.preventDefault();
+                        evt.stopPropagation();
```

Comment:
@itsjeyd just to make sure - without this `stopPropagation` dropping an item using keyboard closed error popup before user could see it, resulting in item being returned to item bank without any explanation why.